presto-product-tests/pom.xml

```diff
@@ -83,8 +83,18 @@
     <build>
         <resources>
             <resource>
-                <directory>${project.basedir}/src/main/resources</directory>
+                <directory>src/main/resources</directory>
                 <filtering>true</filtering>
+                <includes>
+                    <include>presto.env</include>
+                </includes>
+            </resource>
+            <resource>
```

Comment:
We need to run resource plugin with filtering enabled only for the `presto.env` file, and we need to run it for all the other resources with filtering disables. Besides the replacing properties this plugin also does the copy of the resources to the `target`.